plugins/netskope/unit_test/test_get_all_url_list.py

```diff
@@ -0,0 +1,105 @@
+import os
+import sys
+
+sys.path.append(os.path.abspath("../"))
+
+import logging
+from unittest import TestCase, mock
+
+from insightconnect_plugin_runtime.exceptions import PluginException
+from parameterized import parameterized
+
+from icon_netskope.actions.get_all_url_list import GetAllUrlList
+from icon_netskope.actions.get_all_url_list.schema import Output
+from icon_netskope.connection.connection import Connection
+from unit_test.mock import (
+    STUB_CONNECTION,
+    mock_request_200_api_v2,
+    mock_request_400_api_v2,
+    mock_request_401_api_v2,
+    mock_request_403_api_v2,
+    mock_request_500_api_v2,
+    mocked_request,
+)
+
+STUB_EXPECTED_RESPONSE_APPLIED = {
+    "urllists": [
+        {
+            "id": 1,
```

Comment:
Please add mock for multiple items, empty items in the list

Reply:
There is a test for empty list output. It is called "test_get_all_url_list_input_params_ok_no_output"